src/main/kotlin/com/cosmotech/api/rbac/CsmRbac.kt

```diff
@@ -161,8 +161,7 @@ open class CsmRbac(
     logger.debug("RBAC ${rbacSecurity.id} - Verifying $user has permission in ACL: $permission")
     val isAuthorized =
         this.verifyPermissionFromRole(permission, getUserRole(rbacSecurity, user), rolesDefinition)
-    logger.debug(
-        "RBAC ${rbacSecurity.id} - $user has permission $permission in ACL: $isAuthorized")
+    logger.debug("RBAC ${rbacSecurity.id} - $user has permission $permission in ACL: $isAuthorized")
```

Comment:
Remove this change to avoid conflict with RBAC which has change a lot since

Reply:
I did a "rebase main" there's no conflict